probe/kubernetes/client.go

```diff
@@ -242,28 +240,19 @@ func (c *client) clientAndType(resource string) (rest.Interface, interface{}, er
 	case "storageclasses":
 		return c.client.StorageV1().RESTClient(), &storagev1.StorageClass{}, nil
 	case "deployments":
-		return c.client.ExtensionsV1beta1().RESTClient(), &apiextensionsv1beta1.Deployment{}, nil
+		return c.client.AppsV1().RESTClient(), &apiappsv1.Deployment{}, nil
```

Comment:
New Scope versions will not work with `kubernetes < 1.16` ? 
Does it makes sense to have checks for `kubernetes < 1.16` too?